models/nc-news.models.js

```diff
@@ -86,4 +86,23 @@ const updateVoteAtArtcileId = (article_id, newVotes) => {
     })
 }
 
-module.exports = { fetchTopics, fetchEndpoints, fetchArticleById, fetchArticles, fetchCommentsByArticleID, insertComment, updateVoteAtArtcileId };
\ No newline at end of file
+const removeCommentById = (comment_id) => {
+    return db.query(`
+    SELECT * 
+    FROM comments 
+    WHERE comment_id = $1
+    `, [comment_id]).then((result) => {
+        if (result.rows.length === 0) {
```

Comment:
this is a bit overcomplicated. We could return * in the delete query and by the result know if the comment existed